lib/hocon/impl/simple_config.rb

```diff
@@ -107,6 +107,57 @@ def hash
     41 * @object.hash
   end
 
+  def self.find_key_or_null(me, key, expected, original_path)
+    v = me.peek_assuming_resolved(key, original_path)
```

Comment:
Nevermind, the fact that the parameter is called "self" in the upstream is confusing